src/views/Export/ExportedFiles/index.tsx

```diff
@@ -84,6 +84,9 @@ function ExportedFiles(props: Props) {
             setUserExports(response.results);
             setExportCount(response.count);
         },
+        shouldPoll: response => (
+            (activeTab === 'pending' && response?.count && response.count > 0) ? 5000 : -1
+        ),
```

Comment:
Why poll when count is greater than zero? 
Is that the in-progress count?

Reply:
yes, if the active tab is pending and the response is more than 0, then we continuously fetch the data